hatch_build.py

```diff
@@ -101,7 +101,7 @@
         "python-ldap>=3.4.4",
     ],
     "leveldb": [
-        "plyvel>=1.5.1",
+        "plyvel>=1.5.1; sys_platform != 'darwin'",
```

Comment:
Not so sure why this is needed

Reply:
I explained that in the comment - and linked to https://github.com/wbolster/plyvel/issues/114. Plyvel is notoriously difficult to install on MacOS - every time there is an upgrade to a new version of OS (Sequoia in this case) it is brokent for at least few months. You likely did not have this problem and had it installed before because you have not yet upgraded to Sequoia :).

So I think disabling it for MacOS sounds like a good idea - it's not **really** important (and optional) feature of Google provider.

Reply:
I added comment explaining it.